examples/timeline/editing/tooltipOnItemChange.html

```diff
@@ -0,0 +1,81 @@
+<html>
+<head>
+  <title>Timeline | Tooltip on item onUpdateTime Option</title>
+
+  <script src="../../../dist/vis.js"></script>
+  <link href="../../../dist/vis.css" rel="stylesheet" type="text/css" />
+
+  <script src="../../googleAnalytics.js"></script>
+</head>
+
+<body>
+
+<h1>Timeline Tooltip on item onUpdateTime Option</h1>
+
+<h2>With <code>tooltipOnItemUpdateTime: true</code>
+</h2>
+
+<div id="mytimeline"></div>
+
+<script>
+
+  // create groups
+  var numberOfGroups = 10; 
+  var groups = new vis.DataSet()
+  for (var i = 0; i < numberOfGroups; i++) {
+    groups.add({
+      id: i,
+      content: 'Truck&nbsp;' + i
+    })
+  }
+  
+  // create items
+  var numberOfItems = 100;
+  var items = new vis.DataSet();
+  var types = [ 'box', 'point', 'range']
+
+  var itemsPerGroup = Math.round(numberOfItems/numberOfGroups);
+
+  for (var truck = 0; truck < numberOfGroups; truck++) {
+    var date = new Date();
+    for (var order = 0; order < itemsPerGroup; order++) {
+      date.setHours(date.getHours() +  4 * (Math.random() < 0.2));
+      var start = new Date(date);
+
+      date.setHours(date.getHours() + 2 + Math.floor(Math.random()*4));
+      var end = new Date(date);
+
+      items.add({
+        id: order + itemsPerGroup * truck,
+        type: types[Math.floor(3 * Math.random())],
+        group: truck,
+        start: start,
+        end: end,
+        content: 'Order ' + order
+      });
+    }
+  }
+  
+  // specify options
+  var options = {
+    orientation: 'top',
+    stack: true,
+    multiselect: true,
+    maxHeight: 400,
+    start: new Date(),
+    end: new Date(1000*60*60*24 + (new Date()).valueOf()),
+    tooltipOnItemUpdateTime: true,
```

Comment:
I really would like to see the new `template` option here to show how it works. Can you please add an simple example!?